src/Connectors/src/ConnectorBase/ConnectionStringConfigurationSource.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Steeltoe.Connector
+{
+    public class ConnectionStringConfigurationSource : IConfigurationSource
```

Comment:
The ConnectionString Configuration source/provider _could_ be in a separate package, grouped under the other `/Configuration` packages, should it move to a dedicated package? I put it here because it only exists as an entry point to the logic in ConnectorBase